src/bootstrap/bootstrap.py

```diff
@@ -596,6 +596,7 @@ def __init__(self, config_toml="", args=None):
         self.download_url = (
             os.getenv("RUSTUP_DIST_SERVER") or self.stage0_data["dist_server"]
         )
+        self.jobs = self.get_toml("build.jobs") or "default"
```

Comment:
section.value doesn't work here, this always returns `default`. You need `self.get_toml("jobs", section="build")`.